utils/google.js

```diff
@@ -90,6 +92,32 @@ const googleApp = {
  * Exported google util functions
  */
 module.exports = {
+  async getFileFromBucket(googleProject, pathToCredsKeyFile, bucketName, fileName) {
```

Comment:
not good error handling - the fixed function is in my branch https://github.com/uc-cdis/gen3-qa/blob/feat/google-expire/utils/google.js#L158
we can copy it here or just wait for my PR to be merged